src/state/contexts/GlobalState.tsx

```diff
@@ -18,7 +18,7 @@ const initialState: GlobalState = {
     'bLAgYxAdX2Ry-nt6aH2ixgvJXbpsEYm28NgJgyqfs-U',
   ),
   pdnsSourceContract: DEFAULT_PDNS_REGISTRY_STATE,
-  gateway: 'ar-io.dev',
```

Comment:
Im guessing this is from testing?

Reply:
yeah - although we still havent' seemed to solve the issue with ar-io.dev so we may want to just revert until the changes can be implemented